app/uk/gov/hmrc/nrs/retrieval/controllers/NonrepRetrievalController.scala

```diff
@@ -27,10 +29,11 @@ import javax.inject.Singleton
 import scala.concurrent.ExecutionContext
 
 @Singleton()
-class NonrepRetrievalController @Inject()(
+class NonrepRetrievalController @Inject() (
   val nonrepRetrievalConnector: NonrepRetrievalConnector,
-  override val controllerComponents: ControllerComponents)(implicit ec: ExecutionContext)
-    extends BackendController(controllerComponents) {
+  override val controllerComponents: ControllerComponents
+)(using ec: ExecutionContext)
```

Comment:
`(using ec: ExecutionContext)` to
`(using ExecutionContext)`

note this doesn't always work, if there is other values in the arg list then we have to keep the variable name, or I haven't found a way around it.